OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
```

Comment:
I agree that changing to a reference is outside the scope of the PR. Perhaps a middle ground would be to add to the doxygen documentation (after moving it to the header) explaining that this constructor does not take ownership?